api/src/api/app/controller/scenario_controller.py

```diff
@@ -44,8 +44,8 @@
 )
 
 class LookupObject:
-    scenario: Scenario = None
-    model: Model = None
+    scenario: Scenario|None = None
+    model: Model|None = None
```

Comment:
does not belong to this feature. general linting stuff.
```suggestion
    scenario: Scenario = None
    model: Model = None
```